template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/RazorPagesWeb-CSharp/Pages/Account/Manage/Disable2fa.cshtml.cs

```diff
@@ -28,12 +28,12 @@ public class Disable2faModel : PageModel
             var user = await _userManager.GetUserAsync(User);
             if (user == null)
             {
-                throw new ApplicationException($"Unable to load user with name '{User.Identity.Name}'.");
+                throw new ApplicationException($"Unable to load user with id '{_userManager.GetUserId(User)}'.");
```

Comment:
Note, its probably usually the case that UserId will be null for this to have failed.